### PR TITLE
React-Global-2022

### PR DESCRIPTION
**Pull request by lxmarinkovic**

### CODE REVIEW

File: content/community/conferences.md [hunk 1]
```diff
@@ -22,6 +22,11 @@ April 1st, 2022. Amsterdam, The Netherlands
 
 [Website](https://www.reactlive.nl/) - [Twitter](https://twitter.com/reactlivenl)
 
+### < React Global > by Geekle {#react-global-2022}
+April 21st-22nd, 2022 - Online Summitt
```
Comment (harish-sethuraman) [inline content/community/conferences.md:26]:
```suggestion
April 21st-22nd, 2022 - Online Summit
```

Comment (lxmarinkovic) [reply content/community/conferences.md]: yes, it is April 21st-22nd, 2022 - Online Summit

**Comment by lxmarinkovic** [comment]
Hi @harish-sethuraman, please review :D

File: content/community/conferences.md (hunk 1 above)
Comment (harish-sethuraman) [reply content/community/conferences.md]: Can we change it to `April 21-22, 2022 - Online Summit`. I see that older conferences were consistent and let's follow that?

**Comment by lxmarinkovic** [comment]
Yes, ofc 😃

**Review by harish-sethuraman — APPROVED** [review]

**Comment by harish-sethuraman** [comment]
Thanks :)

**Comment by lxmarinkovic** [comment]
@harish-sethuraman can you help me with this one [https://github.com/reactjs/reactjs.org/issues/4272](url)

**Pull request merged**